esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/adding_a_recipe.rst

```diff
@@ -0,0 +1,100 @@
+able to successfully run it with ESMValtool without error.
+
+* Run ESMValTool locally with your recipe, make sure to take note of the memory
+  and time expenditure provided by the terminal output::
+
+    esmvaltool run <your_recipe_name.yml>
+
+* Run the same recipe on JASMIN, again taking note of the memory and time
+  expenditure printed to the terminal.
+
+* Stop any running recipe_test_workflow workflows::
+
+    cylc stop "a_running_recipe_test_workflow"
+
+* Add the recipe to the ``[task parameters]`` section of the |RTW| ``flow.cylc``
+  workflow file (make sure to place it within the "fast" or "medium" category
+  depending on how long it took to run):
+
+* Run the workflow. The process task for the new recipe should succeed, but its
+  compare task should fail with an error that the reference data (KGO) does
+  not exist::
+
+    cylc vip -O <your_site_name>
+
+* Locate the workflow run folder of the workflow you just completed.
+
+* Copy the output files from the "cycle" folder (run/share/bin/cycle) of the
+  workflow you just ran, to your site specific KGO rootpath folder
+  (this folder should be set as the value for the variable "KGO_ROOT_PATH="
+  in your rose-suite<your_site>.conf file)::
+
+    cp -r <directory_of_recipe_output_cycle_folder> <KGO_rootpath_folder>
+
+* Change directory to the rootpath KGO directory::
+
+    cd <KGO_rootpath_directory>
+
+* Allow write permissions for all users on the directory and it's
+  subdirectories of the recipe you've added to the KGO folder::
+
+    chmod a+w <the_directory_of_the_recipe_you_have_copied_into_the_KGO_folder>
+
+* Run the RTW again::
+
+    cylc vip -O <your_site_name>
+
+* The workflow should now succeed.
+
+* Take note of how long the run took  to complete on cylc. This can be found in the `job.time` section of the task listed as `process_<your_recipe>`.
+
+* Locate your local <site>.cylc file, found in (recipe_test_workflow/site).
+
+* Open it in your preferred code editor.
+
+* Locate the `COMPUTE` section, it should look something like this::
+
+    [[COMPUTE]]
+    platform = <your_platform_here>
+    execution time limit = PT2M
+    [[[directives]]]
+    --wckey = RTW
+    --ntasks = {{ MAX_PARALLEL_TASKS }}
+    --mem = 2G
+
+* Compare the `execution time limit` and --mem (memory) units here with the
+  readings you took locally. If your local readings do not exceed these, then
+  you have successfully added the recipe to the workflow, and can now commit
+  and push your changes.
+
+
+* If either of the time/memory readings from your local run are larger than the
+  values specified in the `COMPUTE` section, you need to add your recipe as
+  another `process` similar to::
+
+    [[process<fast=recipe_albedolandcover>]]
+    # Actual: 0m31s, 2.5 GB on 2024-04-08.
+    execution time limit = PT2M
+    [[[directives]]]
+        --mem = 3G
+
+* Variable (fast, medium) must be consistent with flow.cylc.
```

Comment:
If this is now moved into the documentation then that should be specified here.

Reply:
I have re-raised this in https://github.com/ESMValGroup/ESMValTool/pull/3614#discussion_r1644087547 👍 I will resolve this on @chrisbillowsMO's behalf, as discussed 😊